ethereum/client/src/abi.rs

```diff
@@ -31,12 +31,65 @@ sol! {
 
         function getBlockNumber(bytes32 merkle_root) external view returns (uint256);
     }
+
+
+
+
 }
 
+#[cfg(test)]
+sol!(
+    #[sol(rpc)]
+    ProxyContract,
+    "../out/ProxyContract.sol/ProxyContract.json"
+);
+
+#[cfg(test)]
+sol!(
+    #[sol(rpc)]
+    ERC20Mock,
+    "../out/ERC20Mock.sol/ERC20Mock.json"
```

Comment:
Can you generate these `.json`s in the `build.rs` script?

Reply:
no, but can do from Makefile

Reply:
Added some decorators